index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Foobar UI</title>
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/foobar-ui.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700;900&family=Poppins:wght@300;400;600;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/base16/dracula.min.css"
+    />
```

Comment:
You can create one styles.css file, and import all the stylesheets there and add that stylesheet link here, no need to import all of them together in one file

Reply:
Hi Nisha,
As we are writing in plain CSS and also several imports can cause resource downloads, I am not using CSS imports in this case. If you see the 'workflow' folder, there is a bash script to merge all those CSS files. In that way, anyone can use the single CSS file in their link tag for using it.
PS: I will use @imports when I will move this CL from CSS to SASS as @imports in SASS behave differently and do not load resources.